src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1296,22 +1296,21 @@ properties:
             fits_keyword: SPTOFFST
             blend_table: True
           x_offset:
-            title: x offset from pattern starting position
+            title: "[arcsec] X offset in SI ideal coordinate frame"
             type: number
             fits_keyword: XOFFSET
             blend_table: True
           y_offset:
-            title: y offset from pattern starting position
+            title: "[arcsec] Y offset in SI ideal coordinate frame"
             type: number
             fits_keyword: YOFFSET
             blend_table: True
           starting_set:
-            title: Starting 4-point set number
+            title: Starting 4-point set in dither set list
             type: integer
-            enum: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
```

Comment:
The keyword dictionary doesn't have an "enum" for this keyword and states "Possible values are 1-10" (note the absence of 0). I chose for this PR to remove the enum.